### PR TITLE
[WIP]CI Files for opensds

### PR DESCRIPTION
**Pull request by satya-gorli**
**What this PR does / why we need it**:
The PR includes CI Tasks like Static Checks (GoMetaLint) , Unit Testing, Integration Test scripts that will be invoked on PR or on specific label
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
  scripts for Static Check with GoMetalint, UnitTest, Integration Test, e2e Test flows 
**Special notes for your reviewer**:
please review the separation of jobs
**Release note**:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (leonwanghui) [inline Makefile]: Please remove this blank line

File: Makefile [hunk 2]
```diff
@@ -26,13 +27,18 @@ ubuntu-dev-setup:
...
 
-build:osdsdock osdslet osdsapiserver osdsctl
+build:osdsdock osdslet osdsapiserver osdsctl osds_verify osds_unit_test osds_integration_test osds_e2eflowtest_build osds_e2etest_build 
 
 prebuild:
 	mkdir -p $(BUILD_DIR)
 
 .PHONY: osdsdock osdslet osdsapiserver osdsctl docker test protoc
 
+
```
Comment (leonwanghui) [inline Makefile]: Is there any reason why these blank lines been added? If not, please remove them.

File: Makefile (hunk 1 above)
Comment (satya-gorli) [reply Makefile]: Removed blanks in Makefile

**Comment by leonwanghui** [comment]
@satya-gorli Could you design a shell file to download those binary files automatically? I think it's not allowed to embed them directly into the project, because it's too large for users to git clone.

**Pull request closed without merging**